forms-flow-api/src/formsflow_api/resources/application.py

```diff
@@ -470,4 +508,36 @@ def delete(application_id, **kwargs):
             return f"Application was successfully deleted from all DB sources with id: {application_id}", HTTPStatus.OK
         except BusinessException as err:
             current_app.logger.error(err.error)
-            return err.error, err.status_code
\ No newline at end of file
+            return err.error, err.status_code
+
+# @cors_preflight("PATCH, OPTIONS")
+# @API.route("/<int:application_id>/update", methods=["PATCH", "OPTIONS"])
```

Comment:
is this method used anywhere in DJ system?

Reply:
Oops! This was left here by mistake. Removed